### PR TITLE
Using cert-manager to renew Prow certs

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: prow/cluster/cert-manager/tls-ing.yaml [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
+    certmanager.k8s.io/acme-http01-edit-in-place: "true"
     kubernetes.io/ingress.class: "gce"
 spec:
   tls:
   - secretName: prow-tls
+    hosts:
+    - prow.istio.io
   rules:
-  - http:
+  - host: prow.istio.io
```
Comment (hklai) [inline prow/cluster/cert-manager/tls-ing.yaml:18]: do we need others like http://velodrome.istio.io?

**Review by hklai — APPROVED** [review]
/lgtm

File: prow/cluster/cert-manager/cluster-issuer.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  namespace: default
+  # Adjust the name here accordingly
+  name: letsencrypt-prod
+spec:
+  acme:
+    # The ACME server URL
+    server: https://acme-v02.api.letsencrypt.org/directory
+    # Email address used for ACME registration
+    email: sebastienvas@google.com
```
Comment (rkpagadala) [inline prow/cluster/cert-manager/cluster-issuer.yaml:14]: change this istio-engprod@

File: prow/cluster/cert-manager/tls-ing.yaml (hunk 1 above)
Comment (sebastienvas) [reply prow/cluster/cert-manager/tls-ing.yaml:18]: not on the prow cluster. This ingress redirecting Prow services. Velodrome is hosted by k8s, so we have to use a proxy.

File: prow/cluster/cert-manager/cluster-issuer.yaml (hunk 2 above)
Comment (sebastienvas) [reply prow/cluster/cert-manager/cluster-issuer.yaml:14]: istio-engprod does not exist, and google.com google groups cannot be reached from external. @hklai do we have a better group email for this ?

Comment (hklai) [reply prow/cluster/cert-manager/cluster-issuer.yaml:14]: istio-oncall@googlegroups.com?

**Pull request merged**